monkey/monkey_island/cc/services/config_schema/internal.py

```diff
@@ -213,24 +213,17 @@
                     "items": {"type": "string"},
                     "description": "List of SSH key pairs to use, when trying to ssh into servers",
                 },
-            },
-            "smb_service": {
-                "title": "SMB service",
-                "type": "object",
-                "properties": {
-                    "smb_download_timeout": {
-                        "title": "SMB download timeout",
-                        "type": "integer",
-                        "default": 30,
-                        "description": "Timeout (in seconds) for SMB download operation (used in "
-                        "various exploits using SMB)",
-                    },
-                    "smb_service_name": {
-                        "title": "SMB service name",
-                        "type": "string",
-                        "default": "InfectionMonkey",
-                        "description": "Name of the SMB service that will be set up to download "
-                        "monkey",
+                "smb_service": {
+                    "title": "SMB service",
+                    "type": "object",
+                    "properties": {
+                        "smb_download_timeout": {
```

Comment:
Why not move it to a const in smbexec.py? Other exploiters don't have timeouts in config

Reply:
I considered it. For the moment, it's the only thing exercising our ability to send custom options to exploiters. I think it's worth keeping just as a reminder that we can do that for now.